Solutions/Auth0/Data Connectors/Auth0Connector/main.py

```diff
@@ -59,15 +60,12 @@ def main(mytimer: func.TimerRequest):
         pass
     logging.info("new config" + str(config))
     state_manager.post(json.dumps(config))
-    sentinel = AzureSentinelConnector(LOG_ANALYTICS_URI, WORKSPACE_ID, SHARED_KEY, LOG_TYPE, queue_size=1000)
-    for el in events:
-        sentinel.send(el)
```

Comment:
Please check any retry  mechanism in case of  sending to sentinel is failed?

Reply:
Not required